test/integration/Makefile

```diff
@@ -7,7 +7,7 @@ PROJECT_ROOT ?= $(abspath ../..)
 
 CLUSTER_NAME := ccm-integration-test
 VERSION ?= $(shell git describe --always --dirty)
-CCM_IMAGE := gcr.io/cloud-provider-vsphere/vsphere-cloud-controller-manager:$(VERSION)
+CCM_IMAGE := gcr.io/cloud-provider-vsphere/cpi/pr/manager:$(VERSION)
```

Comment:
same comment here for the manifests, fine with changing the url but needs to be reflected in the manifests

Reply:
For sure. I was holding off on that portion of the docs because I was afraid of creating a race condition where the docs point to the new place but the new place isn't working yet. My experience here has been once this PR goes in, it takes an iteration or two (usually 2-3 hours) between this repo and the Prow jobs to get everything working correctly. But that's a pretty narrow window, so it's probably better to just change it now. :)

Reply:
👍